### DISCUSSION TITLE
What is the advantage of  convert HTML and CSS to SVG?What specific scenarios can it be applied to?

### DISCUSSION THREAD

**Discussion by LLipsky** [vercel/satori#298] [Q&A]
What is the advantage of  convert HTML and CSS to SVG?
What specific scenarios can it be applied to?

**Comment by yuzhanglong** [comment]
mark

**Comment by shuding** [comment]
The main use case is to generate dynamic images without the browser environment, but also have some abilities from the browser (e.g. use CSS to layout and position elements). Example: https://vercel.com/blog/introducing-vercel-og-image-generation-fast-dynamic-social-card-images

**Comment by wei** [comment]
One use-case is my project [socialify](https://github.com/wei/socialify) which generates social images for any github project to be included inside README, websites, and beyond. 

Satori saves us the headache of manually laying out components inside svgs but instead, we just need to focus on laying out designs using html/css and converting it to svg/png easily.

[Link][socialify-edit-link] / Live Preview:
![socialify](https://socialify.git.ci/wei/socialify/image?description=1&font=Raleway&issues=1&language=1&pattern=Charlie%20Brown&stargazers=1&theme=Auto)

[socialify-image]: https://socialify.git.ci/wei/socialify/image?description=1&font=Raleway&issues=1&language=1&pattern=Charlie%20Brown&stargazers=1&theme=Auto
[socialify-edit-link]: https://socialify.git.ci/wei/socialify?description=1&font=Raleway&issues=1&language=1&pattern=Charlie%20Brown&stargazers=1&theme=Auto

**Reply by wobsoriano** [reply]
Is it just me getting 401 errors?

**Reply by wei** [reply]
@wobsoriano thanks for the reply, my GitHub token expired. Just fixed~
